src/components/MuiProvider/index.js

```diff
@@ -11,6 +11,20 @@ export const muiTheme = createTheme({
     borderRadius: 10,
   },
   components: {
+    MuiTypography: {
+      styleOverrides: {
+        root: {
+          color: "#000",
+        },
+      },
+    },
+    MuiContainer: {
```

Comment:
Same thing for container to fix the color issue for Next.js link component inside Mui container component.